lib/post_message_definition.yml

```diff
@@ -74,6 +76,9 @@ post_messages:
       oauthError:
         payload:
           <<: *user_session_properties
+          # member_guid:
```

Comment:
Keeping this as a sample of how optional fields will work. This will be uncommented in a future PR.